### DISCUSSION TITLE
Can I output one YAML property in quotes?

### DISCUSSION THREAD

**Discussion by lacarvalho91** [mikefarah/yq#1552] [Q&A]
I've run into a problem which is caused by another system reading `235e732` as a number. It reads from a file that is being written to by `yq`:
```
yq -r '.foo.bar' > bar.txt << EOF
foo:
    bar: 235e732
EOF
```

I then saw in the docs about styles so I tried:
```
yq -r '.foo.bar style="double"' > bar.txt << EOF
foo:
    bar: 235e732
EOF
```

While this does indeed wrap `235e732` in quotes, it also returns the full document. Is there a way to get `yq` to quote one property and only output that property?

**Comment by mikefarah** [comment]
By default `yq` unwraps scalar values (removes quotes) if its just printing the scalar. So you need to do add quotes and tell it not to remove them:

```
yq -r=false '.foo.bar | . style="double"' bar.txt
"235e732"

```
